Server/src/controllers/GraphsController.ts

```diff
@@ -16,8 +16,8 @@ export class GraphsController {
      * @param response 
      * An object containing a response: Response
      */
-    async createRequestForSingleSavedGraph(request: Request, response: Response) {
-        let requestParam = request.params.oneSavedGraph;
+    async createRequestForSingleGraph(request: Request, response: Response) {
+        let requestParam = request.params.graphStateId;

```

Comment:
I dont think we should use params at all. Instead use request.query to make use of key value pairs

Reply:
Using params for single variable passes is best practice. It looks much cleaner and also allows us to differentiate from multiple variable passing, which is where request.query should be used.

Reply:
@GD-Dheer params is the convention for accessing a resource, the query can be mixed up with params, we use params for verbosity https://restfulapi.net/http-methods/ 
![image](https://user-images.githubusercontent.com/22376976/105797693-76d27280-5f5e-11eb-9027-55e55c0fa06c.png)